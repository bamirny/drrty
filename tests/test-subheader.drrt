def int main() {
	makeHeader("Hello There!");
	makeSubheader("My name is Rania");
	makeSubheader("This is a Drrty test!");

	return 0;
	

}