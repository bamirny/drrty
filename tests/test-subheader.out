<h1>Hello There!</h1>
<h2>My name is Rania</h2>
<h2>This is a Drrty test!</h2>
